app/sdil/config/ConnectorWiring.scala

```diff
@@ -38,6 +38,8 @@ trait ConnectorWiring extends CommonWiring {
   lazy val authConnector: AuthConnector = new DefaultAuthConnector(httpClient, serviceconfig)
   lazy val sdilConnector: SoftDrinksIndustryLevyConnector = wire[SoftDrinksIndustryLevyConnector]
   lazy val payApiConnector: PayApiConnector = new PayApiConnector(httpClient, serviceconfig)
+  lazy val directDebitBackendConnector: DirectDebitBackendConnector =
```

Comment:
Can this be `wire`'d instead of explicitly instantiated with `new`?

Reply:
I tried but it got an error saying there are multiple instances of ServiesConfig